### PR TITLE
Refactor Comments display on Giveaways show page

### PR DESCRIPTION
**Pull request by nikolatriki**
Notion task: https://www.notion.so/Refactor-Comments-display-on-Giveaways-show-page-02c6d717f4fe4b21849081924923c248

### CODE REVIEW

**Review by violeta-p — COMMENTED** [review]
The loop should be outside of partial. For each comment, you should render the partial

**Review by violeta-p — COMMENTED** [review]
I'd suggest adding specs here.

**Comment by nikolatriki** [comment]
I need help with the nested object 'comment' test.

**Review by OrceMaceski — APPROVED** [review]

**Review by violeta-p — APPROVED** [review]

File: spec/views/giveaways/show.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
```
Comment (khadzhinov) [inline spec/views/giveaways/show.html.erb_spec.rb:1]: pls follow rubicon

**Pull request merged**